app/templates/_base_page.html

```diff
@@ -30,7 +30,12 @@
 {% endblock %}
 
 {% block footer_support_links %}
-  <a href="/terms-and-conditions" class="terms-and-conditions">Terms and conditions</a>
+  <h2 class="visuallyhidden">Support links</h2>
+    <ul>
+        <li><a href="/terms-and-conditions" class="terms-and-conditions">Terms and conditions</a></li>
+        <li><a href="/cookies">Cookies</a></li>
+        <li>Built by the <a href="https://www.gov.uk/government/organisations/government-digital-service">Government Digital Service</a></li>
```

Comment:
Is there a reason not to include the ` rel="external"` on the link here? Seems odd to have it in the supplier app but not the buyer one?